update tutor (#2716)
helix-editor/helix: runtime/tutor.txt

 Type J to join together lines in selection.

 1. Move the cursor to the line below marked -->.

>  2. Type x four times or 4x to join the sentence to one line.

Reviewer: ```suggestion
 2. Type x four times or 4x to select all four lines.
```
